apps/ui/src/api/useGetApplication.ts

```diff
@@ -48,7 +50,19 @@ const useGetApplication = (id?: string | number) => {
 				throw error;
 			}
 
-			return await response.json();
+			return await response.json().then((data: ApplicationResponseData) => {
+				// Filter out data if they contain null values and application meta data
+				if (data.contents) {
+					const fields = Object.entries(data.contents).reduce((acc, [key, value]) => {
+						if (value !== null && key !== 'applicationId' && key !== 'createdAt' && key !== 'updatedAt') {
+							acc[key as keyof ApplicationContentsResponse] = value;
```

Comment:
Could potentially add a type guard similar to [apps/ui/src/components/pages/application/utils/validatorFunctions.ts](https://github.com/Pan-Canadian-Genome-Library/daco/pull/239/files#diff-2eb49e26c00f20d6648f6d928902173d3a5d24bda8d50f36076fa01bcf7cd180) to avoid using `as`